### PR TITLE
Add copy link after token been sent for the user doesnt have email set up

### PR DESCRIPTION
**Pull request by peterhashair**
### Description:

Add copy link after the token has been sent. 

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/UsersManager/API.php [hunk 1]
```diff
@@ -793,6 +795,15 @@ public function inviteUser($userLogin, $email, $initialIdSite = null, $expiryInD
          * @param string $email The new user's e-mail.
          */
         Piwik::postEvent('UsersManager.inviteUser.end', [$userLogin, $email]);
+
+        /**
+         * return invite link, in case someone doesn't have email enable
```
Comment (justinvelluppillai) [inline plugins/UsersManager/API.php]:
```suggestion
         * return invite link, in case someone doesn't have email enabled
```

**Comment by justinvelluppillai** [comment]
@peterhashair I've just taken a look to see if I can progress this in your absence but I can't see it's including many of @Javi-Ormaechea's UI suggestions and not clear what compromises have been made or why. 

The designs he provided have adjusted headings, moved info text into better locations, include modals (I understand maybe this isn't possible but based on the technical requirements maybe he could suggest a better way that is possible rather than us guessing at UI), and different flows.

Also the success message right now always seems to say "the invite has been resent to user" which I don't think is correct.

**Comment by peterhashair** [comment]
@justinvelluppillai just double check, so we want the design popup up workflow, and also move the `copy link` to notification 
image

**Comment by justinvelluppillai** [comment]
@peterhashair ideally as close as possible to @Javi-Ormaechea's design as possible. If there are technical limitations (ie we don't store the token) then discussing how this would impact the design with @Javi-Ormaechea would be the next step so that he can come up with a new design that is technically possible and serves the UI best.

**Pull request closed without merging**